lib/publiccloud/provider.pm

```diff
@@ -514,11 +519,11 @@ sub terraform_apply {
             my $vpc_security_group_ids = script_output("aws ec2 describe-security-groups --region '" . $self->provider_client->region . "' --filters 'Name=group-name,Values=tf-sg' --query 'SecurityGroups[0].GroupId' --output text");
             my $availability_zone = script_output("aws ec2 describe-instance-type-offerings --location-type availability-zone  --filters Name=instance-type,Values=" . $instance_type . "  --region '" . $self->provider_client->region . "' --query 'InstanceTypeOfferings[0].Location' --output 'text'");
             my $subnet_id = script_output("aws ec2 describe-subnets --region '" . $self->provider_client->region . "' --filters 'Name=tag:Name,Values=tf-subnet' 'Name=availabilityZone,Values=" . $availability_zone . "' --query 'Subnets[0].SubnetId' --output text");
-            my $ipv6_address_count = get_var('PUBLIC_CLOUD_EC2_IPV6_ADDRESS_COUNT', 1);
+            my $ipv6_prefix_count = get_var('PUBLIC_CLOUD_EC2_IPV6_PREFIX_COUNT', 0);
```

Comment:
can you elaborate on this change ?  Why we renaming variable ? Why it is EC2 specifc as now we already have IPv6 for google and amazon ?

Reply:
This has been discarded. I will elaborate on this in [poo#138197 - Enable IPv6 in EC2 or GCE](https://progress.opensuse.org/issues/138197)